packages/ember-auto-import/ts/auto-import.ts

```diff
@@ -208,14 +208,16 @@ export default class AutoImport implements AutoImportSharedAPI {
   // have their own webpack-generated hashes and (2) the runtime loader code
   // can't easily be told about broccoli-asset-rev's hashes.
   private configureFingerprints(host: AppInstance) {
-    let pattern = 'assets/chunk.*.js';
+    let patterns = ['assets/chunk.*.js', 'assets/chunk..*.css'];
```

Comment:
```suggestion
    let patterns = ['assets/chunk.*.js', 'assets/chunk.*.css'];
```
I think the extra `.` isn't needed here?

Reply:
as chunk files named as `chunk.[some hash here].css` seem actually legit? 🤔

Reply:
We were running into this issue in one of our apps so I tried adding the pattern from this PR in our own config but it didn't work. Removing the double `.` next to the `*` made it work as expected.

The chunks look like this `/assets/chunk.536.b58262bb39d5d25c6454.css` so I think it either needs to be the pattern I suggested (which removes the double `.`) or something like `assets/chunk.*.*.css` which is a more strict match but I would go for consistency with the js pattern anyways.